packages/cli/src/Generate.ts

```diff
@@ -104,6 +104,8 @@ ${bold('Examples')}
       '--data-proxy': Boolean,
       '--accelerate': Boolean,
       '--no-engine': Boolean,
+      // https://github.com/prisma/prisma/issues/22513
```

Comment:
This is really relevant?

Reply:
56bde3bd4ca1f716cf6c337f73c7ce5ec39577b7